content/docs/setup/kubernetes/install/helm/index.md

```diff
@@ -136,24 +111,25 @@ $ kubectl -n istio-system edit deployment istio-galley
 
     {{< /warning >}}
 
-1. Uninstall steps:
+1. Verify that all `58` Istio CRDs were committed to the Kubernetes api-server using the following command:
 
     {{< text bash >}}
-
-    $ kubectl delete namespace istio-system
-
+    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
+    58
     {{< /text >}}
 
-1. If desired, run the following command to delete all CRDs:
-
-    {{< warning >}}
-    Deleting CRDs permanently deletes any configuration changes that you have made to Istio.
-    {{< /warning >}}
+1. Render and apply Istio's core components. For a production environment use the recommended Helm values or customize as needed:
 
     {{< text bash >}}
+    $ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system | kubectl apply -f -
+    {{< /text >}}
 
-    $ kubectl delete -f $HOME/istio-fetch/istio-init/files
+    {{< tip >}}
```

Comment:
Why is this a tip instead of the last step?

Reply:
Because it is a special case of customization for the bookinfo. It would help for you to run to run the bookinfo app and notice it does not work with the default values. Other people who need to test this would benefit from this PR getting in sooner.